general/general.md

```diff
@@ -11,3 +11,11 @@ It takes sometime to sync the progress. If it doesnt sync for a long time, ask f
 ### Q3. Why I am facing "Nothing to compile" issue while compiling my contract in Windows Device?
 
 You need to check and Update your `Hardhat` Version to "2.9.6". You can do this by entering this command ` npm install --save-dev "hardhat@^2.9.6" `
+
+### Q4. How can I get the most out of the learn web 3 platform?
+
+lw3 is so much more than a bunch of tutorials. The projects are very easy to follow and if you allow it will just spook feed you the code, but you learn nothing like that. Here are the best ways to make the most out of all the tracks.
+
+- Personalise every project in some way. You can change the look and feel to make it your own iterate and add some extra functionality to the UI/contract/both. This is ideal if you want to use the projects in your portfolio as you are differentiating yourself from everyone else.
+- Go down plenty of rabbit holes. If you are new to web3, which most people here are then you are going to be bombarded with new tech and phrases, make sure to click through all the links and DYOR as much as possible to ensure you are understanding what you are learning. The nature of these tutorials and projects is to give you just enough to get something working, but if you really want to build skills and a career in web3 then you need to go deeper.
```

Comment:
"do your own research" instead of DYOR as people might not know what that means

Reply:
done